toolboxes/modules/cfpdes/partials/introduction.adoc

```diff
@@ -233,7 +233,29 @@ u_i = g_i(x,t,\mu), \quad i=1,\dots,N
 ++++
 ****
 
-The user provides the expression for stem:[(g_i)_{i=1\dots N}] in the `.json` file  only if Dirichlet conditions are used.
+The user provides the expression for stem:[(g_i)_{i=1\dots N}] in the `.json` file only if Dirichlet conditions are used.
+
+.Examples of Dirichlet boundary condition
+[.examp#ex:boudaryconditions:dirichlet]
+****
+[source, json]
+----
+"Dirichlet": <1>
+{
+    "Gamma0": <2>
+    {
+        "markers": ["Bottom","Right","Bottom-Left","Top-Right"], <3>
+        "expr": "0" <4>
+    }
+}
+----
+****
+
+<1> Keyword for Dirichlet conditions.
```

Comment:
replace with `<.>` it will follow the order above

Reply:
done in c6df538df2ccf52fdc470f6dcb74c3c178b767d8